src/test/java/com/jcraft/jsch/AlgorithmsIT.java

```diff
@@ -474,7 +474,7 @@ private void doSftp(Session session, boolean debugException) throws Exception {
       session.disconnect();
       jschAppender.stop();
       // Sleep for a short bit to allow logs from test container to catch up
-      Thread.sleep(100L);
+      Thread.sleep(250L);
```

Comment:
The other way to solve this would be to remove the assertions checking the OpenSSH logs from the Docker container.

It seems that the logs from the Docker container are async and may be fully posted by the time the tests start checking them.